firebase-config/src/test/java/com/google/firebase/remoteconfig/internal/PersonalizationTest.java

```diff
@@ -74,8 +82,7 @@ public void setUp() {
 
     doAnswer(invocation -> FAKE_LOGS.add(invocation.getArgument(2)))
         .when(mockAnalyticsConnector)
-        .logEvent(
-            eq(ANALYTICS_ORIGIN_PERSONALIZATION), eq(ANALYTICS_PULL_EVENT), any(Bundle.class));
+        .logEvent(eq(ANALYTICS_ORIGIN_PERSONALIZATION), anyString(), any(Bundle.class));
```

Comment:
Could us the `or` operator from [`AdditionalMatchers`](https://www.javadoc.io/doc/org.mockito/mockito-core/latest/org/mockito/AdditionalMatchers.html) to get either of `ANALYTICS_PULL_EVENT` or `ANALYTICS_PULL_EVENT_INTERNAL`.